client/packages/invoices/src/Returns/api/hooks/line/useDeleteSelectedInboundLines.ts

```diff
@@ -53,16 +64,3 @@ export const useDeleteSelectedInboundReturnLines = (): (() => void) => {
 
   return confirmAndDelete;
 };
-
-const useDeleteInboundLines = () => {
-  const { invoiceNumber = '' } = useParams();
-  const queryClient = useQueryClient();
-  const api = useReturnsApi();
-  const queryKey = api.keys.inboundDetail(invoiceNumber);
-
-  // TODO: Replace with actual mutation
-  // return useMutation(api.updateInboundReturnLines, {
-  return useMutation(async (_lines: InboundReturnLineFragment[]) => {}, {
-    onSettled: () => queryClient.invalidateQueries(queryKey),
-  });
-};
```

Comment:
Seemed unnecessary to make *another* hook when this is just calling an existing hook (updateLines) with a specific value (number of packs = 0)